Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -507,6 +507,10 @@ def genGrid(data1, data2, gm, deep=True, grid=None, geo=None):
         vg.SetPoints(geopts)
     else:
         vg = grid
+    # Add a GlobalIds array to keep track of cell ids throughout the pipeline
+    globalIds = numpy_to_vtk_wrapper(numpy.arange(0, vg.GetNumberOfCells()), deep=True)
```

Comment:
@danlipsa why we have to do this? Meaning why we have to explicitly append the global ids?